opentelemetry-exporter-google-cloud/src/opentelemetry/exporter/cloud_trace/__init__.py

```diff
@@ -364,6 +364,27 @@ def _extract_resources(resource: Resource) -> Dict[str, str]:
     }
 
 
+LABELS_MAPPING = {
+    # https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/exceptions.md#attributes
+    "exception.message": "/error/message",
+    "exception.type": "/error/name",
+    "exception.stacktrace": "/stacktrace",
+    # this one might be http.flavor? I'm not sure
+    "http.scheme": "/http/client_protocol",
+    "http.host": "/http/host",
+    "http.method": "/http/method",
+    # https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/http.md#common-attributes
+    "http.request_content_length": "/http/request/size",
+    "http.response_content_length": "/http/response/size",
+    "http.route": "/http/route",
+    "http.status_code": "/http/status_code",
+    "http.url": "/http/url",
+    "http.user_agent": "/http/user_agent",
+    # https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/resource/semantic_conventions/process.md#process
+    "process.pid": "/pid",
```

Comment:
Same with this one, it is not in attributes but in `span.resource`

Reply:
removed as well